app/webpacker/components/SearchWidget/MultiSearchInput.jsx

```diff
@@ -93,18 +109,27 @@ export default function MultiSearchInput({
       // FUI's dropdown will automatically remove selected items from the
       // options left for selection.
       fetchJsonOrError(url(debouncedSearch))
-        .then(({ data }) => setResults(data.result.map(itemToOption)))
+        .then(({ data }) => {
+          switch (model || '') {
```

Comment:
Veto for this switch block as well. The search results should be structured in a way that the existing iterator can access them without having to change frontend code.

Reply:
Removed